doc/data-walloon.rst

```diff
@@ -1,2 +1,4 @@
 Walloon-specific Data
 =====================
+
+* ``data/walloon/custom_costs_rc.csv`` – custom cost assumptions used by the Walloon configuration. Data provided by ICEDD.
```

Comment:
add that data/custom_powerplants.csv was updated for Nuclear, for a list of countries, ... and where data was taken from? Default nuclear was removed